docs/faq.rst

```diff
@@ -228,6 +228,32 @@ Scylla comes with its own version of the Apache Cassandra client tools, in the p
 
 We recommend uninstalling Apache Cassandra before installing :code:`scylla-tools`.
 
+.. _faq-pinning:
+
+Can I install or upgrade to a patch release other than latest on Debian or Ubuntu?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The APT package manager used for Ubuntu, Debian, and image installations of ScyllaDB installs the latest patch 
+release (x.y.z) of a given major release (x.y). To remain on a ScyllaDB patch release that is not the latest, you can 
+use pinning as a workaround.
+
+The following example shows pinning ScyllaDB Enterprise version 2021.1.0-0.20210511.9e8e7d58b-1:
+
+.. code-block:: console
+
+   $ cat <<EOF | sudo tee /etc/apt/preferences.d/99scylla-enterprise
+   Package: scylla-enterprise*
+   Pin: version 2021.1.0-0.20210511.9e8e7d58b-1
+   Pin-Priority: 1001
+   EOF
+
+Pinning may be particularly useful when you want to downgrade ScyllaDB or upgrade to a version that is not the latest 
+available version.
+
+See `this topic <https://unix.stackexchange.com/questions/350192/apt-get-not-properly-resolving-a-dependency-on-a-fixed-version-in-a-debian-ubunt>`_ 
```

Comment:
well not sure if we should cross ref to non official docs(that just state what is said here), below ubuntu community is enough imho

also above is a convenience, so other option is to name ALL packages with their respective version, which is tedious and error prone (but our upgrade guides do use it for revert)

Reply:
Thanks, I've applied your suggestions. 
I've removed the link and added the alternative solution suggested by @avi in [this comment](https://github.com/scylladb/scylla-enterprise/issues/2176#issuecomment-1095041407), as it's less complicated than listing all the dependencies with full names.